Running Windows having this type of error

D:\Portfolio\Project\auto_localization_maker\windows\flutter\generated_plugin_registrant.cc(13,3): error C3861: 'nb_utils_pluginRegisterWithRegistrar': identifier not found [D:\Portfolio\Project\auto_localization_maker\build\windows\runner\auto_localization_maker.vcxproj]
Exception: Build process failed.

--- a/lib/src/utils/decorations.dart
+++ b/lib/src/utils/decorations.dart
@@ -132,6 +132,7 @@ Decoration boxDecorationRoundedWithShadow(
   double? blurRadius,
   double? spreadRadius,
   Offset offset = const Offset(0.0, 0.0),
+  LinearGradient? gradient,
 }) {
   return BoxDecoration(
     boxShadow: defaultBoxShadow(
@@ -141,6 +142,7 @@ Decoration boxDecorationRoundedWithShadow(
       offset: offset,
     ),
     color: backgroundColor,
+    gradient: gradient,
     borderRadius: radius(radiusAll.toDouble()),
   );
 }

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -31,4 +31,4 @@ flutter:
       macos:
         pluginClass: NbUtilsPlugin
       windows:
-        pluginClass: nb_utils_plugin
+        pluginClass: NbUtilsPlugin

